.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+	"trailingComma": "es5",
```

Comment:
Do we really need this? (Not a huge fan of this cuz it is annoying at times) :P

Reply:
Well, prettier will do it for you and it's better for diffs